packages/core/src/render3/interfaces/definition.ts

```diff
@@ -316,6 +322,11 @@ export interface ComponentDef<T> extends DirectiveDef<T> {
    */
   tView: TView|null;
 
+  /**
+   * A function added by the {@see ɵɵStandaloneFeature} and used by the framework to create standalone injectors.
+   */
+  getStandaloneInjector: ((parentInjector: EnvironmentInjector) => EnvironmentInjector | null)|null;
```

Comment:
nit: should it be optional (not `null`) from typing perspective (as non-standalone would *not* have this field at all)?
```suggestion
  getStandaloneInjector?: ((parentInjector: EnvironmentInjector) => EnvironmentInjector | null);
```

Reply:
Good idea, but the definition fields are all required for a reason - we want the definition objects to all be the same shape and thus have monomorphic property access wherever they're used. Using `null` instead of `undefined` helps enforce this.

Reply:
Ok, I see that we assign `null` as a value there by default (thus keeping the object monomorphic) 👍
(we'd need to adjust the `ngContentSelectors` field type in a followup PR)